examples/sklearn_penguins/sklearn_predict_extractor_test.py

```diff
@@ -139,7 +139,7 @@ def testMakeSklearnPredictExtractorWithMultiModels(self):
       def check_result(actual):
         try:
           for item in actual:
-            self.assertEqual(item['labels'].shape, item['predictions'].shape)
+            self.assertEqual(len(item['labels']), len(item['predictions']))
```

Comment:
Please don't modify files in other projects without approval of the project owners.  Check the CODEOWNERS to see who owns each project.